packages/player/scripts/test.sh

```diff
@@ -1,2 +1,38 @@
 export "$(grep -vE "^(#.*|\s*)$" .env)"
-TEST_USER=$TEST_USER npm run wtr $1
+# TEST_USER=$TEST_USER npm run wtr $1
+# Event Producer seems to prevent WTR from running all tests in one go, so we need to run them one by one
```

Comment:
Web Test Runner wouldn't run normally but running each test individually worked, so I succumbed to this hack. 

I believe Event Producer is the culprit, doesn't seem it's made for testing in mind. After my first attempts of running the tests I got its IDB filled with 300 000 items and Chrome hang on memory errors. Also got "Crash: Failed to execute 'postMessage' on 'MessagePort': Data cannot be cloned, out of memory.", I suppose EP tried sending those 300k events over the MessagePort too... Tried adding proper cleanups of the IDBs after each test but it still refused to run. :(